src/main/java/tool/MainProgram.java

```diff
@@ -32,7 +32,7 @@ private MainProgram() {
     public static void main(String[] args) {
         //Just use this project for now (src), will have to change
         //to the target project with the gradle stuff
-        startAnalyse(new String[] {"src"});
+
```

Comment:
Det här håller jag inte med om. Varför är den borttagen?

Reply:
We used main to test the program during development before we had real tests written, accidentaly removed startAnalyse, reverted this change in next commit